ServiceWebsite/ServiceWebsite.Services/KinlyPlatformService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using ServiceWebsite.Common;
+using ServiceWebsite.Common.Security;
+using ServiceWebsite.Domain;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace ServiceWebsite.Services
+{
+    public class KinlyPlatformService : IKinlyPlatformService
+    {
+        private readonly ICustomJwtTokenProvider _customJwtTokenProvider;
+
+        public KinlyPlatformService(ICustomJwtTokenProvider customJwtTokenProvider)
+        {
+            _customJwtTokenProvider = customJwtTokenProvider;
+        }
+
+        public async Task<TestCallResult> GetTestCallScoreAsync(Guid participantId)
+        {
+            HttpResponseMessage responseMessage;
+            using (var httpClient = new HttpClient())
+            {
+                var requestUri = $"https://dev.self-test.hearings.hmcts.net/api/v1/testcall/{participantId}";
```

Comment:
this string should be in secret file "https://dev.self-test.hearings.hmcts.net/api/v1/